src/VisualStudio/IntegrationTest/TestUtilities/Common/Reference.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using Roslyn.Utilities;
+
+namespace Microsoft.VisualStudio.IntegrationTest.Utilities.Common
+{
+    /// <summary>
+    /// Represents a result of a Find References operation.
+    /// </summary>
+    [Serializable]
+    public class Reference : IEquatable<Reference>
+    {
+        public string FilePath { get; set; }
```

Comment:
These need to be serializable, so it's that or write a custom serializer. I'm not convinced the latter is justified in this case.